From: Chmouel Boudjnah <chmouel@mandrakesoft.com>
Subject: Cardmgr
To: Juan Quintela <quintela@mandrakesoft.com>,
	Mandrake Kernel <kernel@linux-mandrake.com>
Date: 21 Nov 2001 14:50:16 +0100


Can you aplly this patch, displaying information about cardmgr is
wrong since it obsolete :

--- linux/drivers/pcmcia/cs.c.old   Tue Nov 20 05:50:53 2001
+++ linux/drivers/pcmcia/cs.c   Wed Nov 21 14:35:27 2001
@@ -90,7 +90,7 @@
 #define OPTIONS PCI_OPT CB_OPT PM_OPT
 #endif
 
-static const char *release = "Linux Kernel Card Services " CS_RELEASE;
+static const char *release = "Linux Kernel Card Services Kernel Version";
 static const char *options = "options: " OPTIONS;
 
 MODULE_AUTHOR("David Hinds <dahinds@users.sourceforge.net>");

